app/index.pug

```diff
@@ -1,9 +1,11 @@
 html
   include shared-views/head
   body(ng-app='nijelApp')
-    include shared-views/nav
+    div(ng-if ="!isAdminState")
+      include shared-views/nav
     main
       ui-view(class='main-view' ng-cloak)
-      include shared-views/footer
+        div(ngif="!isAdminState")
+        include shared-views/footer
```

Comment:
just curious, for the nav element, it has a parent div, with the isAdminState applied to it, and for the footer the div is a sibling to the footer element, rather not the parent